### ISSUE TITLE
Constructor of Ticker is not suitable for XRP

### ISSUE THREAD

**Issue by hunghd60115** [bitbankinc/java-bitbankcc#1]
Exception: 
```bash
com.fasterxml.jackson.databind.exc.InvalidFormatException: Can not construct instance of java.lang.Integer from String value '26.999': not a valid Integer value
 at [Source: {"success":1,"data":{"sell":"26.999","buy":"26.480","high":"29.791","low":"25.100","last":"26.480","vol":"1313122.8847","timestamp":1508308082912}}; line: 1, column: 22] (through reference chain: cc.bitbank.entity.response.TickerResponse["data"]->cc.bitbank.entity.Ticker["sell"])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:55)
	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:797)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._parseInteger(StdDeserializer.java:401)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:289)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:271)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:538)
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:106)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:242)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:118)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:538)
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:106)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:242)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:118)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3051)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2146)
	at cc.bitbank.deserializer.JsonDecorder.decode(JsonDecorder.java:17)
	at cc.bitbank.Bitbankcc.httpExecute(Bitbankcc.java:132)
	at cc.bitbank.Bitbankcc.doHttpGet(Bitbankcc.java:150)
	at cc.bitbank.Bitbankcc.getTicker(Bitbankcc.java:171)
```

Current constructor: 
```java
public class Ticker extends Data {
    public int sell;
    public int buy;
    public int high;
    public int low;
    public int last;
    public BigDecimal vol;
    public Date timestamp;
.....
}
```

But in case XRP value is xx.xxx
 so this value can not parse to int.

Please help me fix this problem. 

Best Regards,
HungHD

**Comment by junderw** [comment]
Thanks for the report. One of our engineers will fix the issue ASAP.

**Comment by junderw** [comment]
I think we will switch to BigDecimal. Hold on.

**Comment by junderw** [comment]
Fixed in d4b663dcd8fc36f1492d973525b20b2b8ddca4a8
